server/appbackend/api.py

```diff
@@ -109,76 +115,52 @@ def post(self, request, *args, **kwargs):
 
 class ConfirmTransferRequestApi(generics.UpdateAPIView):
     queryset = TransferRequest.objects.all()
-    serializer_class = TransferRequestSerializer
+    serializer_class = []
```

Comment:
what does empty array mean here?

Reply:
It means that the request body will be empty.